uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss

```diff
@@ -0,0 +1,36 @@
+.page-civicrm-leaveandabsences-dashboard {
+  h1.page-title,
+  #branding {
+    display: none !important;
+  }
+
+  #page {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.admin-dashboard-page {
```

Comment:
all the styling contained here seem overrides of the Shoreditch style. Why are they necessary?